schemas/component.json

```diff
@@ -0,0 +1,135 @@
+{
+	"title": "JSON schema for Eightshift blocks used in component manifest.json",
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"$schema": {
+			"type": "string"
+		},
+		"componentName": {
+			"type": "string",
+			"pattern": "^[^\/][a-z0-9-]*$",
+			"description": "The name for a componentName is a unique string that identifies a componentName."
+		},
+		"title": {
+			"type": "string",
+			"description": "This is the display title for your component."
+		},
+		"componentClass": {
+			"type": "string",
+			"description": "Set the value your component class will have."
+		},
+		"description": {
+			"type": "string",
+			"description": "This is a short description for your block, which can be translated with our translation functions. This will be shown in the block inspector."
```

Comment:
```suggestion
			"description": "A short description for your block that can be translated with our translation functions. This will be shown in the block inspector."
```